contribs/gnodepgraph/depgraph.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
+	"github.com/gnolang/gno/gnovm/pkg/gnomod"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	osm "github.com/gnolang/gno/tm2/pkg/os"
+)
+
+type depGraphCfg struct {
+	verbose        bool
+	rootDir        string
+	output         string
+	multipleGraphs bool
+}
+
+func newDepGraphCmd(io commands.IO) *commands.Command {
+	cfg := &depGraphCfg{}
+
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "gnodepgraph",
+			ShortUsage: "gnodepgraph [flags] <package> [<package>...]",
+			ShortHelp:  "generates dependency graphs for the specified packages",
+		},
+		cfg,
+		func(_ context.Context, args []string) error {
+			return execDepGraph(cfg, args, io)
+		},
+	)
+}
+
+func (c *depGraphCfg) RegisterFlags(fs *flag.FlagSet) {
+	rootDir := gnoenv.RootDir()
+	fs.BoolVar(&c.verbose, "v", false, "verbose output when lintning")
```

Comment:
would be nice to have a non-graphviz format by default, could be just lines with:

		A -> B
		A -> C
		C -> D

something we can easily read and grep.

then, the graphviz format would be a flag.